### PR TITLE
Update documentation, release 3.13.0

### PR DESCRIPTION
**Pull request by hroncok**
Related to https://github.com/fedora-python/crypt_r/issues/6

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -156,6 +181,18 @@ check it against the original:
...
        raise ValueError("hashed version doesn't validate against original")
 
+--------------
+
+
+Changelog
+---------
+
+For historical changes when this module was included in Python,
+please refer to the `Python 3.12 Changelog`_.
```
Comment (hroncok) [inline README.rst:191]: This link does not seem to work, not sure why :/

Comment (hroncok) [reply README.rst:191]: Oh, it was a missing `_` in the link below.

**Comment by hroncok** [comment]
Rendered at https://test.pypi.org/project/crypt-r/3.12.3/

**Comment by hroncok** [comment]
I think this is ready to be published. I don't like how Test PyPI renders the license file in the license field, but I *think* we are not doing it wrong.

**Comment by befeleme** [comment]
> I think this is ready to be published. I don't like how Test PyPI renders the license file in the license field, but I _think_ we are not doing it wrong.

We aren't. We could declare `license.text` and put the SPDX expression there which would probably be easier to digest for people looking at the PyPI page. I *think* setuptools would still include the license file if we don't declare the `license.file` key in pyproject.toml, per https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html:

> All files specified by the license-files configuration parameter in pyproject.toml and/or equivalent in setup.cfg/setup.py; note that if you don’t explicitly set this parameter, setuptools will include any files that match the following glob patterns: LICENSE*, LICENCE*, COPYING*, NOTICE*, AUTHORS**;

**Comment by befeleme** [comment]
Other than that, the actual changes contained in this PR make sense to me, AFAICT all the suggestions for the changelog are there, render is also alright.

**Comment by hroncok** [comment]
> > I don't like how Test PyPI renders the license file in the license field, but I _think_ we are not doing it wrong.
> 
> We aren't. We could also declare `license.text` and put the SPDX expression there which would probably be easier to digest for people looking at the PyPI page...

I don't think it's worth it. Do you?

**Comment by befeleme** [comment]
Not really, the benefit is microscopic.

**Pull request merged**